exchanges/ftx/ftx_wrapper.go

```diff
@@ -430,12 +430,21 @@ func (f *FTX) UpdateOrderbook(ctx context.Context, p currency.Pair, assetType as
 	if err != nil {
 		return book, err
 	}
+
+	// Bear tokens have illiquid books and contain negative place holders.
+	bearToken := strings.Contains(p.String(), "BEAR")
```

Comment:
Is there ever any instance of desiring a negative order quantity in an order book? From the looks of things BEAR is showing up because they have such terrible order books, but something else later could also have a terrible order book and that this bug fix might be too specific. 

What I'm getting at is that we can save the string conversion and inspection for every REST orderbook and just ignore negative sizes in general, unless there are scenarios I don't know of.

Reply:
> Is there ever any instance of desiring a negative order quantity in an order book?

Not that I have come across. 

> something else later could also have a terrible order book and that this bug fix might be too specific.

Yeah, you are correct, but this bug, for now, is specific to the Bear books, which should be fixed. I don't want to suppress or filter any errors, especially for a spot book, I would rather error out for the moment and not be able to trade on that liquidity because the entire book could be compromised. 

> we can save the string conversion and inspection for every REST orderbook and just ignore negative sizes in general,
You make a good point about doing a string check for every book request, being a REST snapshot it wasn't the top of my priority for speed considerations. But what I can do, is check to see if there are negative amounts first then do a string comparison on both book sides.

Reply:
I'm happy with the negative check first 🎉